### PR TITLE
add GitHub Student Developer Pack page

### LINKED ISSUE #76: Create the "GitHub Student Developer Pack" page
Previous website source: https://github.com/gitpod-io/website/blob/master/src/pages/github-student-developer-pack.tsx
Previous website link: https://www.gitpod.io/github-student-developer-pack/

Note: There is no Figma design. Please follow the previous website guideline with the new branding applied.

### PR DESCRIPTION
**Pull request by nisarhassan12**
fixes #76 

This is how it looks: 

image

### CODE REVIEW

File: src/contents/github-student-developer-pack.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    duration: "Per User/Month",
+    features: [
+      "100 hours/month",
+      "Private & Public Repos",
+      "4 Parallel Workspaces",
+      "30min Timeout",
+      "Limited to 6 months",
+    ],
+    btnText: "Claim Offer",
+    btnHref: "https://gitpod.io/subscription/",
```
Comment (gtsiolis) [inline src/contents/github-student-developer-pack.ts:24]: **issue**: Minor link  update.

```suggestion
    btnHref: "https://gitpod.io/plans/",
```

File: src/contents/github-student-developer-pack.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    price: isEurope() ? "€8" : "$9",
+    duration: "Per User/Month",
+    features: [
+      "All in personal plus",
+      "16 Parallel Workspaces",
+      "Team Manageable",
+      "1h Timeout",
+    ],
+    btnText: "Buy Now",
+    btnHref: "https://gitpod.io/subscription/",
```
Comment (gtsiolis) [inline src/contents/github-student-developer-pack.ts:37]: **issue**: Minor link  update.

```suggestion
    btnHref: "https://gitpod.io/plans/",
```

File: src/components/github-student-developer-pack/header.svelte [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+<header>
+  <h1>GitHub Student Developer Pack</h1>
+  <p>
+    Make Your Life Easier with Gitpod. With Gitpod you have no more tedious
+    setups, you save hours of compiling code, and you can start coding from any
+    device, immediately.
+  </p>
+  <a target="_blank" href="https://gitpod.io/plans" class="btn-conversion">
+    Claim Offer
```
Comment (gtsiolis) [inline src/components/github-student-developer-pack/header.svelte:15]: **question**: Is this sufficient for claiming the offer? Is there anything we need to also change on the product side within `/plans`? /cc @jankeromnes

**Review by gtsiolis — APPROVED** [review]
Thanks for adding this @nisarhassan12! 🎉 

Left two minor comments for some deprecated links we changed during the dashboard redesign and one question. ❓

**Comment by ChristinFrohne** [comment]
Great, thanks Nisar! Some small comments:

- I remember that they required to have a link to their GitHub Student Developer Pack page. (https://education.github.com/pack/?utm_source=github+gitpod)
Please keep the link just like on the former page

- I feel we should highlight the free plan (this is what the GitHub pack is all about) and not trick them into the standard student plan. 

- we should highlight how much money they are saving compared to the usual pricing 
Bildschirmfoto 2021-04-29 um 10 28 35

**Review by ChristinFrohne — APPROVED** [review]
Looks good now, thanks!

**Pull request merged**